com.unity.render-pipelines.universal/Editor/ShaderGUI/BaseShaderGUI.cs

```diff
@@ -145,6 +149,7 @@ public virtual void FindProperties(MaterialProperty[] properties)
         {
             surfaceTypeProp = FindProperty("_Surface", properties);
             blendModeProp = FindProperty("_Blend", properties);
+            preserveSpecProp = FindProperty("_PreserveSpecular", properties, false);    // TODO: should be mandatory?
```

Comment:
Create a discussion about this so we're not leaving a TODO for something that can be decided before this PR lands. :)